web/package.json

```diff
@@ -75,6 +75,7 @@
     "@visx/group": "^3.3.0",
     "@visx/shape": "^3.5.0",
     "@visx/text": "^3.3.0",
+    "clipboard": "^2.0.11",
```

Comment:
I would avoid adding a package for this if we can. Take a look at https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard instead.

Reply:
Thanks for the suggestion! Since we are hacking we are just trying things out as quickly as possible :) But if you guys implement this in production then it makes sense to build it nicer!